### PR TITLE
Show file size for uplink ls command

### PR DESCRIPTION
**Pull request by mniewrzal**

### CODE REVIEW

File: cmd/uplink/cmd/ls.go [hunk 1]
```diff
@@ -93,7 +93,7 @@ func listFiles(ctx context.Context, bs buckets.Store, u *url.URL, prependBucket
 	startAfter := paths.New("")
 
 	for {
-		items, more, err := o.List(ctx, paths.New(u.Path), startAfter, nil, *recursiveFlag, 0, meta.Modified)
```
Comment (mniewrzal) [inline cmd/uplink/cmd/ls.go]: I'm not sure why it's not working with `meta.Modified|meta.Size`. I will debug it.

Comment (kaloyan-raev) [reply cmd/uplink/cmd/ls.go]: I think the issue is in the Stream Store.

Comment (kaloyan-raev) [reply cmd/uplink/cmd/ls.go]: Pushed a fix to this PR.

Comment (mniewrzal) [reply cmd/uplink/cmd/ls.go]: Thanks :)

**Pull request merged**